### PR TITLE
Add "Principal Engineer" deputy project lead

### PR DESCRIPTION
**Pull request by tacaswell**
Nominate Antony Lee to the position.

There is also an incidental change to the spelling of "API Consistency Lead"

### CODE REVIEW

File: deputy_project_leads.md [hunk 1]
```diff
@@ -38,6 +38,20 @@ This include checking that:
   the existing functions
 
 
+## Principle Engineer
+
+Matplotlib relies on wide and deep code base to implement its public API; the
+low-level details need to be correct to faithfully implement that API.
+In contrast to the API Consistency Leader, who is responsible for what
+the library does, the Principle Engineer is responsible for how. They
+are the point of contact for:
```
Comment (jklymak) [inline deputy_project_leads.md:47]: I have learned many principles from @anntzer, but regardless suggest we nominate him for "Principal Engineer"  

```suggestion
## Principal Engineer

Matplotlib relies on wide and deep code base to implement its public API; the
low-level details need to be correct to faithfully implement that API.
In contrast to the API Consistency Leader, who is responsible for what
the library does, the Principal Engineer is responsible for how. They
are the point of contact for:
```

File: deputy_project_leads.md [hunk 2]
```diff
@@ -38,6 +38,20 @@ This include checking that:
   the existing functions
 
 
+## Principle Engineer
```
Comment (anntzer) [inline deputy_project_leads.md:41]: The correct spelling is *principal*.  See e.g. https://www.dictionary.com/e/principal-vs-principle/ :)

File: deputy_project_leads.md [hunk 3]
```diff
@@ -38,6 +38,20 @@ This include checking that:
...
+
+Matplotlib relies on wide and deep code base to implement its public API; the
+low-level details need to be correct to faithfully implement that API.
+In contrast to the API Consistency Leader, who is responsible for what
+the library does, the Principle Engineer is responsible for how. They
```
Comment (anntzer) [inline deputy_project_leads.md:46]: ditto

File: people.md [hunk 4]
```diff
@@ -9,7 +9,8 @@ Thomas Caswell is the current Project Lead
 These are the current deputy project leader positions:
 
 - **Release Mangagers** :  (3.3, 3.4) : Elliott Sales de Andrade
-- **API consistency Leader** : Tim Hoffmann
+- **API Consistency Leader** : Tim Hoffmann
+- **Principle Engineer**: Antony Lee
```
Comment (anntzer) [inline people.md:13]: ditto

**Comment by tacaswell** [comment]
I swear English is my native language...

**Review by efiring — APPROVED** [review]

File: people.md [hunk 5]
```diff
@@ -9,7 +9,8 @@ Thomas Caswell is the current Project Lead
 These are the current deputy project leader positions:
 
 - **Release Mangagers** :  (3.3, 3.4) : Elliott Sales de Andrade
-- **API consistency Leader** : Tim Hoffmann
+- **API Consistency Leader** : Tim Hoffmann
+- **Principal Engineer**: Antony Lee
 - **Reference Documentation Leader** :
```
Comment (timhoffm) [inline people.md:14]: You could put my name here. AFAIR I'm taking the role until somebody else steps up.

**Review by timhoffm — APPROVED** [review]

File: people.md (hunk 5 above)
Comment (tacaswell) [reply people.md:14]: I will do that in another PR to keep these easier to audit later.

File: deputy_project_leads.md [hunk 6]
```diff
@@ -38,6 +38,20 @@ This include checking that:
   the existing functions
 
 
+## Principal Engineer
+
+Matplotlib relies on wide and deep code base to implement its public API; the
```
Comment (dopplershift) [inline deputy_project_leads.md:43]: "on *a* wide and deep code base"

**Review by dopplershift — APPROVED** [review]
Minor grammar nitpick.

**Review by anntzer — APPROVED** [review]
Thanks all :)

**Pull request merged**